### ISSUE TITLE
Crashlog timestamp format should be unix epoch number, not string (breaking Java client)

### ISSUE THREAD

**Issue by HSerg** [appium/appium#6785]
## The problem

``` java
driver.manage().logs().get("crashlog");
```

produce

```
java.lang.ClassCastException: java.lang.String cannot be cast to java.lang.Long
    at org.openqa.selenium.remote.RemoteLogs.getRemoteEntries(RemoteLogs.java:88)
    at org.openqa.selenium.remote.RemoteLogs.get(RemoteLogs.java:77)
    at ru.sklabs.appium.crashLogTest.main(crashLogTest.java:39)
```
## Environment
- Appium  1.5.3
- Mac OS X 10.11.6
- Java-client 3.4.1
- Xcode 7.3.1
- iOS Simulator: CoreSimulator 209.19 - Device: iPhone 5s - Runtime: iOS 8.4 (12H141) - DeviceType: iPhone 5s
## Appium logs

```
[HTTP] --> POST /wd/hub/session/dcfc7ff7-0529-4aa6-8d57-8a52a2b5d94a/log {"type":"crashlog"}
[MJSONWP] Calling AppiumDriver.getLog() with args: ["crashlog","dcfc7ff7-0529-...
[debug] [iOS] Executing iOS command 'getLog'
[debug] [iOS] Retrieving 'crashlog' logs
[MJSONWP] Responding to client with driver.getLog() result: [{"timestamp":"2016-08-30T1...
[HTTP] <-- POST /wd/hub/session/dcfc7ff7-0529-4aa6-8d57-8a52a2b5d94a/log 200 20 ms - 55271 
```
## Details

``` java
Object raw = driver.getExecuteMethod().execute(DriverCommand.GET_LOG, ImmutableMap.of("type", "crashlog"));
List<Map<String, Object>> rawList = (List<Map<String, Object>>) raw;
System.out.println(rawList);
```

produce

`[{level=ALL, message=..., timestamp=2016-08-30T11:33:05.000Z}]`

**Comment by mykola-mokhnach** [comment]
It might be similar to https://github.com/appium/appium/issues/4835

**Comment by artgo** [comment]
+1

**Comment by artgo** [comment]
@mykola-mokhnach related but different. Here on Simulator it fails to get crash logs...

**Comment by jlipps** [comment]
The response from Appium looks OK in the logs. I would raise this as an issue at https://github.com/appium/java-client

**Comment by HSerg** [comment]
Looks OK, but violates selenium api. "Timestamp" value must be "_UNIX Epoch timestamp at which this log entry was created_".

**Comment by jlipps** [comment]
@HSerg interesting, I think you're right. Will reopen and get into our backlog.

**Comment by HSerg** [comment]
It can fix - https://github.com/appium/appium-ios-driver/pull/212

**Comment by jlipps** [comment]
Thanks, fixed in https://github.com/appium/appium-ios-driver/pull/212!

**Comment by artgo** [comment]
Thank you for closing, @jlipps 
This is a blocker issue for us. When do you think it can get to a release?

**Comment by imurchie** [comment]
It's available in 1.6.4 beta (install with `npm uninstall -g appium && npm install -g appium@beta`)

**Comment by artgo** [comment]
@isbjorn FYI
